### PR TITLE
fix(validation regex): ensure RFC-5322 compliance

### PR DESCRIPTION
**Pull request by jbueza**
http://tools.ietf.org/html/rfc5322

### CODE REVIEW

**Comment by fengthedroid** [comment]
:+1:

**Comment by stryderjzw** [comment]
:+1:

**Comment by BelfordZ** [comment]
TY for the PR
Please submit a failing test first please!

**Comment by jbueza** [comment]
@BelfordZ fixed! There's also more information here: http://www.regular-expressions.info/email.html

**Comment by BelfordZ** [comment]
@jbueza just added CI.
Can you rebase and Ill merge this puppy?

**Comment by jbueza** [comment]
@BelfordZ roger that

File: src/spec/email.js [hunk 1]
```diff
@@ -8,16 +8,20 @@ var shouldBeTrue = [
   "firstname.lastname@domain.com",
   "email@subdomain.domain.com",
   "firstname+lastname@domain.com",
-  "email@123.123.123.123",
   "1234567890@domain.com",
   "email@domain-one.com",
   "_______@domain.com",
   "email@domain.name",
   "email@domain.co.jp",
+  "あいうえお@domain.com",
```
Comment (jbueza) [inline src/spec/email.js:16]: dat dere unicode

**Pull request merged**